docs/modules/dev/partials/domain/core-intro.adoc

```diff
@@ -0,0 +1,6 @@
+The xref:dev:domain/core/index.adoc[core component]s of James are:
+
+ * xref:domain/core/server.adoc[Server]
```

Comment:
I am unsure that the "server" is a core component in itself, but rather the entity that assembles "core components" together.

IMO **mailQueue**, **taskManager**, **mailRepository**, **mailetContainer** makes more sense.

Reply:
Thanks! These are exactly the kinds of comments I am hoping to get.

Would you mind making this comment in [the ongoing email thread](https://www.mail-archive.com/server-dev@james.apache.org/msg65859.html)?

I'd really like some clarity before moving forward. 😀

Reply:
Sure

Reply:
Answered ;-)